internal/precache/precache-workload/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.hub.docker.com/library/golang:1.20 as builder
```

Comment:
There is a red hat one. See the PR from Michail

Reply:
Changed to `registry.access.redhat.com/ubi9/go-toolset:1.20`